sdk/search/Azure.Search/src/Utilities/SearchExtensions.cs

```diff
@@ -47,5 +48,25 @@ public static string GetSearchServiceName(this Uri endpoint)
                 host.Substring(0, separator) :
                 null;
         }
+
+        /// <summary>
+        /// Join a collection of strings into a single comma separated string.
+        /// If the collection is null or empty, a null string will be returned.
+        /// </summary>
+        /// <param name="items">The items to join.</param>
```

Comment:
Could actually take an `IEnumerable<T>` and be more permissive. Just use `Count()` instead (it's optimized for `ICollection/ICollection<T>`).

Reply:
The only users are internal `IList<T>` so I'm not going to bother yet.   If anyone else needs this, then yes, let's use Enumerable's Count which does those type specific optimizations.